### ISSUE TITLE
Cat neutral in the middle

### ISSUE THREAD

**Issue by Andrzej-Andrzej** [strengejacke/sjPlot#869]
Is it possible to place neutral category in the middle of Likert scale on the plot_likert where 0 line is or will it always be situated on the side of the plot ?

**Comment by strengejacke** [comment]
Yes, I decided to put it on the side of the plot, because I have no idea how the neutral category could be placed in the middle. Say, we have 10% neutral. Would this mean you have both 5% to the positive and negative side (which adds up to 10%, but would not represent the neutral votes for each direction). Or 10% positive and negative? But then we have 20% neutral.

**Comment by Andrzej-Andrzej** [comment]
That makes sense, now I can understand the logic behind it thank you very much, one more thing, is it possible to do coord.flip() whem plotting Likert bars, something like horizontal = FALSE in HH::likert package ?
[https://www.r-bloggers.com/2012/12/plotting-likert-scales/](url)
